src/react/databrowser/buckets/BucketCreate.jsx

```diff
@@ -78,50 +87,61 @@ function BucketCreate(props) {
         );
     };
 
-    return <FormContainer>
-        <div className='sc-title'> create  bucket </div>
-        <fieldset>
-            <label htmlFor='userName'> Name </label>
-            <Input
+    const selectLocations = useMemo(() => {
+        return locationWithIngestion(locations, capabilities);
+    }, [locations, capabilities]);
+
+    return <Form innerRef={formRef}>
+        <F.Title> create new bucket </F.Title>
+        <F.Fieldset>
+            <F.Label tooltipMessages={['Must be unique']}>
+                Name
+            </F.Label>
+            <F.Input
                 type='text'
                 id='name'
                 name='name'
-                placeholder='User Name'
-                onChange={onChange}
-                value={bucket.name}
+                innerRef={register}
+                onChange={clearServerError}
                 autoComplete='off' />
-        </fieldset>
-        <fieldset>
+            <F.ErrorInput id='error-name' hasError={errors.name}> {errors.name?.message} </F.ErrorInput>
+        </F.Fieldset>
+        <F.Fieldset>
```

Comment:
Suggestion: you can add a tooltip with the following content `The Storage Location cannot be changed after creation`